### DISCUSSION TITLE
How to create a new content collection similar to blogs?

### DISCUSSION THREAD

**Discussion by divya-binil** [arthelokyo/astrowind#322] [Q&A]
I am using Astrowind template. I have two content sections, one is blogs and the other one is a list of patents. I could list the blogs correctly, but i am finding it difficult to include the patents. Should I replicate all the folder and files of Blog folder inside the "src/pages" to create patents listing similarly.

**Comment by Dacong-wu** [comment]
Hello, you can find the answer here [span-blog](https://github.com/Dacong-wu/astro-blog/tree/main/src/pages/%5B...photography%5D)
I created a collection of images

**Comment by prototypa** [comment]
Hello @divya-binil. We are moving this to Discussion because it is not an issue with **AstroWind**.
